From 21e9de4bc079d8159831a4a0eccac2b06d6b2db8 Mon Sep 17 00:00:00 2001
From: Christian Hesse <mail@eworm.de>
Date: Mon, 27 Oct 2014 23:27:32 +0100
Subject: [PATCH 1/1] fix udev rule

Do not set MODE and GROUP but give access to useres with console access.
---
 70-u2f.rules | 6 +++++-
 1 file changed, 5 insertions(+), 1 deletion(-)

diff --git a/70-u2f.rules b/70-u2f.rules
index c7152fe..ab436f1 100644
--- a/70-u2f.rules
+++ b/70-u2f.rules
@@ -1 +1,5 @@
-KERNEL=="hidraw*", SUBSYSTEM=="hidraw", MODE="0664", GROUP="plugdev", ATTRS{idVendor}=="1050", ATTRS{idProduct}=="0113|0114|0115|0116|0120"
+ACTION!="add|change", GOTO="u2f_end"
+
+KERNEL=="hidraw*", SUBSYSTEM=="hidraw", ATTRS{idVendor}=="1050", ATTRS{idProduct}=="0113|0114|0115|0116|0120", ENV{ID_SECURITY_TOKEN}="1"
+
+LABEL="u2f_end"
-- 
2.1.2

